src/docfx/restore/index/DependencySlotPool.cs

```diff
@@ -111,7 +111,7 @@ public static (string path, T slot) AcquireSlot(string url, LockType type, Func<
                             var existed = false;
                             foreach (var i in slots)
                             {
-                                if (DateTime.UtcNow - i.LastAccessDate > TimeSpan.FromSeconds(_defaultLockdownTimeInSecond))
+                                if (!i.Restored || DateTime.UtcNow - i.LastAccessDate > TimeSpan.FromSeconds(_defaultLockdownTimeInSecond))
```

Comment:
>!i.Restored [](start = 36, length = 11)

the slot maybe right being used by another docfx-restore?

Reply:
no, one index file can be only access by one consumer at the same time